### PR TITLE
Adding a basic module to display spotify current song;

### PR DESCRIPTION
**Pull request by guilbep**

### CODE REVIEW

File: py3status/modules/spotify.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+This module displays the current "artist - title" playing in Spotify.
+
+Last modified: 2015-03-26
+Author: Pierre Guilbert <pierre@1000mercis.com>
+License: GNU GPL http://www.gnu.org/licenses/gpl.html
```
Comment (ultrabug) [inline py3status/modules/spotify.py]: please accept and change to BSD license

File: py3status/modules/spotify.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+"""
+
+from time import time
+import dbus, dbus.proxies
+
+class Py3status:
+
+    # available configuration parameters
+    cache_timeout = 0
```
Comment (ultrabug) [inline py3status/modules/spotify.py]: having a format placeholder to let users configure a **format** parameter would be great ! can you do this plz ?

see the https://github.com/ultrabug/py3status/blob/master/py3status/modules/net_rate.py module for an example of format and placeholder usage

**Comment by ultrabug** [comment]
awesome mate, I made a few comments plz :)

File: py3status/modules/spotify.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        bus = dbus.SessionBus()
+
+        try:
+            self.__bus  = bus.get_object('com.spotify.qt', '/')
+            self.player = dbus.Interface(self.__bus, 'org.freedesktop.MediaPlayer2')
+
+            title = self.player.GetMetadata().get('xesam:title')
+            artist = self.player.GetMetadata().get('xesam:artist')[0]
+            album = self.player.GetMetadata().get('xesam:album')
+            return title + " by " + artist
```
Comment (Zopieux) [inline py3status/modules/spotify.py]:
```python
return self.format.format(title=title, artist=artist, album=album)
```

File: py3status/modules/spotify.py (hunk 2 above)
Comment (Zopieux) [inline py3status/modules/spotify.py]:
```python
format = "{title} by {artist}"
```

**Comment by guilbep** [comment]
Thank you @Zopieux It did helped; @ultrabug I wanted to add support for clicks adding the on_click method to the module.. but py3status was not liking it (meaning crashing!!). Maybe I've got a wrong version. I let you know in a separate issue! Thank you all.

**Comment by ultrabug** [comment]
Thanks @guilbep, two last remarks if I may :
- would you please accept and change the license to BSD ?
- the default format is rather **long** don't you think ? Maybe a simple {artist} : {title} would be enough ?

**Comment by ultrabug** [comment]
Thx a lot mate !

**Pull request merged**